src/test/kotlin/JiraPerformanceComparisonIT.kt

```diff
@@ -55,32 +50,6 @@ class JiraPerformanceComparisonIT {
         dumpMegaSlowWaterfalls(results)
     }
 
-    @Test
-    fun shouldOnlyProcessGatheredData() {
```

Comment:
Let's squash the PR

Reply:
Oops, I meant that we should squash the two "process only" commits (first in JPCIT, second in a separate IT). But the independent lifespan change got squashed in too. Unsquashing and restitching (`git reset`, `git add -p`) is worth the effort in general, but with our current deadlines it's ok to leave it oversquashed.